### PR TITLE
Make it publishable

### PR DESCRIPTION
**Pull request by kdisneur**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,11 +1,138 @@
...
+  [{:kaur, "~> 0.1.0"}]
+end
+```
+
+## Usage
+
+### `:ok`, `:error` tuples A.K.A `Result` tuples
+
+`{:ok, value}` and `{:error, reason}` is a common pattern in Erlang and Elixir. The `Kaur.Result` module add functions
+which help dealing with this values without getting out of your pipeline.
```
Comment (Fenntasy) [inline README.md]: `these`

File: README.md [hunk 2]
```diff
@@ -1,11 +1,138 @@
...
+
+You can have a look to the documentation to know what's available.
```
Comment (Fenntasy) [inline README.md]: Can we have a link to it in this sentence?

File: README.md [hunk 3]
```diff
@@ -1,11 +1,138 @@
...
-  [{:kaur, github: "fewlines/kaur"}]
+  [{:kaur, "~> 0.1.0"}]
+end
+```
+
+## Usage
+
+### `:ok`, `:error` tuples A.K.A `Result` tuples
+
+`{:ok, value}` and `{:error, reason}` is a common pattern in Erlang and Elixir. The `Kaur.Result` module add functions
```
Comment (RomSarto) [inline README.md]: The `Kaur.Result` module adds functions

File: README.md (hunk 1 above)
Comment (RomSarto) [inline README.md]: which help deal with these values

File: README.md [hunk 4]
```diff
@@ -1,11 +1,138 @@
...
+defmodule Person do
+  defstruct [:name, :age, :bonus]
 end
+
+defmodule MyModule do
+  alias Kaur.Result
+
+  def rent_a_car(person) do
+    person
+    |> validate_age
```
Comment (Fenntasy) [inline README.md]: `validate()` for credo

File: README.md (hunk 2 above)
Comment (RomSarto) [inline README.md]: have a look at

File: README.md [hunk 5]
```diff
@@ -1,11 +1,138 @@
...
+### `:ok`, `:error` tuples A.K.A `Result` tuples
+
+`{:ok, value}` and `{:error, reason}` is a common pattern in Erlang and Elixir. The `Kaur.Result` module add functions
+which help dealing with this values without getting out of your pipeline.
+
+You can have a look to the documentation to know what's available.
+
+Just below you will find a small example of how your code could look like using `Kaur.Result`. In this example we try
+to determine if a person can rent a car. People can rent a car if they are between 21 and 99 year old and have a bonus
+greater than 0.8.
```
Comment (RomSarto) [inline README.md]: Maybe also show what your code would look like without Kaur?

**Review by Fenntasy — CHANGES_REQUESTED** [review]

**Review by Fenntasy — APPROVED** [review]
Would it be useful to mention that they can see examples of Kaur.Result inside of Kaur.Environment?

**Review by xvw — APPROVED** [review]

File: README.md [hunk 6]
```diff
@@ -1,11 +1,160 @@
+[![Hex pm](https://img.shields.io/hexpm/v/kaur.svg)](https://hex.pm/packages/kaur)
+[![Build
+Status](https://travis-ci.org/fewlinesco/kaur.svg?branch=master)](https://travis-ci.org/fewlinesco/kaur)
+[![Inline docs](http://inch-ci.org/github/fewlinesco/kaur.svg)](http://inch-ci.org/github/fewlinesco/kaur)
+
 # Kaur
 
-A bunch of helper functions we use in different projects.
+A bunch of helper functions which ease the development of your applications.
```
Comment (tgautier) [inline README.md]: I suggest we replace `which` by `to`. Don't ask my why, it just sounds better.

```diff
- A bunch of helper functions which ease the development of your applications.
+ A bunch of helper functions to ease the development of your applications.

```

File: README.md [hunk 7]
```diff
@@ -1,11 +1,160 @@
...
 
 ## Installation
 
 ```elixir
 def deps do
-  [{:kaur, github: "fewlines/kaur"}]
+  [{:kaur, "~> 0.1.0"}]
```
Comment (tgautier) [inline README.md]: Isn't the first version of `kaur` going to be `1.0.0`?

**Review by ninjarab — APPROVED** [review]

File: mix.exs [hunk 8]
```diff
@@ -1,22 +1,43 @@
...
 
+  @project_url "https://github.com/fewlinesco/kaur"
+
   def project do
     [app: :kaur,
-     version: "0.1.0",
-     elixir: "~> 1.5",
      build_embedded: Mix.env == :prod,
+     deps: deps(),
+     description: "A bunch of helper functions which ease the development of your applications",
```
Comment (tgautier) [inline mix.exs]: Same as above, I propose we replace `with` by `to`

File: README.md [hunk 9]
```diff
@@ -1,11 +1,160 @@
+[![Hex pm](https://img.shields.io/hexpm/v/kaur.svg)](https://hex.pm/packages/kaur)
+[![Build
+Status](https://travis-ci.org/fewlinesco/kaur.svg?branch=master)](https://travis-ci.org/fewlinesco/kaur)
+[![Inline docs](http://inch-ci.org/github/fewlinesco/kaur.svg)](http://inch-ci.org/github/fewlinesco/kaur)
+
 # Kaur
```
Comment (tgautier) [inline README.md:6]: How about adding a small note about the pronunciation?

The way they do in dictionaries etc...

```
> Pronounced |kɔː|
```

File: README.md (hunk 7 above)
Comment (kdisneur) [reply README.md]: as you wish.

File: README.md [hunk 10]
```diff
@@ -1,11 +1,162 @@
...
+  [{:kaur, "~> 1.0.0"}]
+end
+```
+
+## Usage
+
+### `:ok`, `:error` tuples A.K.A `Result` tuples
+
+`{:ok, value}` and `{:error, reason}` is a common pattern in Erlang and Elixir. The `Kaur.Result` module adds functions
+which help deal with these values without getting out of your pipeline.
```
Comment (tgautier) [inline README.md]: Oh I didn't get that one on the first pass. I also suggest we replace `which` by `to`.

**Review by tgautier — APPROVED** [review]

**Comment by kdisneur** [comment]
Yeahhh : 1.0 is coming 🚀 

Thanks for the review

**Pull request merged**